validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Loan, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V525Spec extends LarEditCheckSpec with BadValueUtils {
+  property("HOEPA Type must = 1, or 2") {
```

Comment:
it was arbitrarily chosen, that's good point I hadn't thought of, I'll change it to status.